src/pages/iou/ReceiptSelector/index.js

```diff
@@ -76,9 +77,27 @@ const defaultProps = {
     },
 };
 
+/**
+ *
+ * @param {Object} file
+ * @param {Object} iou
+ * @param {String} iouType
+ * @param {String} reportID
+ * @param {Object} report
+ */
+function setReceiptAndNavigate(file, iou, iouType, reportID, report) {
```

Comment:
Since this relays on parameters that are available in `ReceiptSelector` it may be better to define this function inside that component.